backend/src/main/java/de/metanome/backend/result_receiver/ResultPrinter.java

```diff
@@ -36,6 +36,16 @@
  */
 public class ResultPrinter implements CloseableOmniscientResultReceiver {
 
+  public static final String RESULT_TEST_DIR = "results/test";
```

Comment:
maybe use enum for endings end enumMap to map them to their String representation